modules/openapi-generator/src/main/resources/JavaVertXWebServer/README.mustache

```diff
@@ -0,0 +1,14 @@
+{{^hideGenerationTimestamp}}
+Project generated on : {{generatedDate}}
+{{/hideGenerationTimestamp}}
+
+## Getting Started
+
+This document assumes you have maven available.
+
+To build the project using maven, run:
+```bash
+mvn package && java -jar target/target/java-vertx-web-rx-server-1.0.0-SNAPSHOT-fat.jar
```

Comment:
Do we need to use {{artifactVersion}} instead of hard-coded `1.0.0-SNAPSHOT`?

Reply:
my mistake, done